migrate/20250110_add_kubernetes_models.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:kubernetes_cluster) do
```

Comment:
Jeremy's PR will likely be merged before this one. We should add a unique constraint for the project_id, location, and name.

https://github.com/ubicloud/ubicloud/pull/2602

Reply:
Merged

Reply:
included